metrics/src/main/java/cucumber/metrics/regulator/module/SpeedRegulatorModule.java

```diff
@@ -0,0 +1,49 @@
+package cucumber.metrics.regulator.module;
```

Comment:
Shouldn't this be placed in the `guice` sub-project as it is a Guice-specific functionality (I can see someone adding similar support for Spring, etc).